packages/documentation/docs/controls/_dropdown_styleguide.md

```diff
@@ -0,0 +1,43 @@
+Dropdowns are containers into which you can insert elements. The dropdown containers do not function independently; they must be opened via a dropdown button, for example. The dropdown container contains items that offer the user a specific selection. These items can be customized depending on their intended use.
+
+Caution, only the dropdown container and its items are described here! For information about the element that opens it, read the article about the dropdown button or the split button.
+
+![Overview](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2353-2278&mode=design&t=OVHeXvLZYLkP2CzN-4)
+
+1. Dropdown Container
+2. Item
+3. Header
+4. Quick Actions
+5. Submenu Item
+6. Checked
+7. Separator
+8. Icon
+9. Label
+
+## Options
+
+- **Header:** Add an header for the dropdown container. 
+- **Quick Actions:** Add a quick action bar.
+- **Checked:** Mark selected items in the dropdown with a check mark.
+- **Submenu:** Add an submenu for a multi-level dropdown.
+- **Separator** Add a separator to visually divide items from each other 
+- **Icon:** Select, if required, a fitting icon for the item.
+- **Label:** Set a label for the dropdown item. We typically use short labels including verbs.
+
+Note: For a date selection dropdown, use the date dropdown!
+
+![Dropdown Examples](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2372-2696&mode=design&t=OVHeXvLZYLkP2CzN-4)
+
+## States
+Dropdown containers themselves have no defined stats - but their items have five: Default, hover, active, disabled and focused. In an active state, dropdown submenu items show an additional level with selectable options.
+
+![Item States](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=2343-42235&mode=design&t=OVHeXvLZYLkP2CzN-4)
+
+## Does and Don’ts
+- Do use dropdowns to showcase related actions
+- Don’t use too many dropdown items - we recommend 7 +/- 2
+
+## Related patterns
+- [Dropdown button](dropdown-button.md)
```

Comment:
since these patterns are in a folder, you need to add them to your path (I believe buttons/dropdown-button.md)